Automattic/wp-calypso
# Blaze on Woo Express: Adding new filter UI for woo products

Task:
Tumblr/a8c-dsp/issues/781

Related to #

Blaze on Woo Express

## Proposed Changes

* In this PR I'm adding the new UI for product filtering on the blaze dashboard for a Woo Store.

## Testing Instructions



* Open calypso and select a Woo Store
* In the left panel select Marketing and then Blaze for Woo Commerce.
* You'll see the available products in the right. Look for the search bar and see that there's a Segmented Control with the options "Products" and "Posts & Pages"
* Toggle between the two options and make sure they are changing 


## Screenshots

Web:

<img width="1124" alt="Screenshot 2024-01-10 at 17 54 51" src="https://github.com/Automattic/wp-calypso/assets/5014402/0b6a66f9-85fc-4186-ba5e-21935753d24f">


Mobile:

![Screenshot 2024-01-10 at 2 19 10 p m](https://github.com/Automattic/wp-calypso/assets/5014402/734f6d73-76e7-46a7-82b8-794875ff1cbf)

Empty State message (Translators):

![Screenshot 2024-01-15 at 11 19 38 a m](https://github.com/Automattic/wp-calypso/assets/5014402/f5e11b20-11e0-4388-b8ad-54d350224b80)




## Pre-merge Checklist
